Dockerfile

```diff
@@ -263,4 +263,7 @@ COPY setup/phpunit/problem-matcher.json /etc/laminas-ci/problem-matcher/phpunit.
 
 RUN useradd -ms /bin/bash testuser
 
+# Copy ubuntu setup
+COPY setup/ubuntu /
```

Comment:
This will copy the files with the ownership of the current user. If that needs to be root you might want to 
```suggestion
COPY --chown=root:root setup/ubuntu /
```